### DISCUSSION TITLE
Wanderer's Space

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#8254] [Q&A] [answered]
Can someone explain (in detail) how to talk to the Wanderers? I checked the wiki, but I still don't know how to get a translator.

**Comment by warp-core** [comment, marked as answer]
If you are playing on continuous, you need to finish Hai Reveal.
If you are playing on 0.9.16.1 or earlier, you ned to have met the Hai and then landing on the Wanderer capital (Vara K'chrai) and you'll get a mission that should point you in the right direction.

**Reply by SultanRTG** [reply]
Thanks, I landed and the story triggered.
